src/main/java/com/spotinst/sdkjava/model/ApiElastigroupListStatefulInstancesDevices.java

```diff
@@ -0,0 +1,78 @@
+package com.spotinst.sdkjava.model;
+
```

Comment:
Add example class for stateful instance API usage since we will be adding more stateful APIs